packages/service-provider-server/src/cli-service-provider.ts

```diff
@@ -112,13 +112,15 @@ class CliServiceProvider implements ServiceProvider {
       // eslint-disable-next-line no-empty
     } catch (e) {
     }
+
     const connectInfo = getConnectInfo(
-      this.uri,
+      this.uri ? this.uri : '',
```

Comment:
We should perhaps add a test for an empty uri in `service-provider-core/src/connect-info.spec.ts`, just to make sure the rest of the information from this function comes out ok.

Reply:
Good call - added a test. I also noticed we didn't have a `test` script in the `service-provider-core` package. Is that intended? I added one, commented below.